package.json

```diff
@@ -17,23 +17,23 @@
         "registry": "https://registry.npmjs.org"
     },
     "dependencies": {
-        "@mojaloop/central-services-logger": "8.1.2",
+        "@hapi/hapi": "18.4.1",
+        "@mojaloop/central-services-logger": "10.4.0",
         "boom": "7.3.0",
-        "@hapi/hapi": "18.4.0",
         "hapi-openapi": "1.2.4"
     },
     "devDependencies": {
         "faucet": "0.0.1",
-        "npm-audit-resolver": "^2.1.0",
-        "npm-check-updates": "3.2.1",
-        "nyc": "14.1.1",
+        "npm-audit-resolver": "^2.2.0",
```

Comment:
caret on dependency

Reply:
I don't think we should worry about these in `devDependencies`

Reply:
i do think for consistency it should be removed, will remove any confusion